bin/zeppelin-daemon.sh

```diff
@@ -190,6 +190,7 @@ function start() {
   wait_zeppelin_is_up_for_ci
   sleep 2
   check_if_process_is_alive
+  echo "ZEPPELIN_CLASSPATH: ${ZEPPELIN_CLASSPATH}" >> "${ZEPPELIN_LOGFILE}"
```

Comment:
I think printing entire classpath i.e `$ZEPPELIN_CLASSPATH_OVERRIDES:$CLASSPATH` is more make sense. what do you think?